base/string_utils.cpp

```diff
@@ -408,6 +408,15 @@ std::string to_string_dac(double d, int dac)
   return ss.str();
 }
 
+std::string to_string_width(long l, int width)
```

Comment:
Should it be a template that accepts all integer types and fails for float types? Or should floating point numbers also be supported?

Reply:
Refactored function to accept any integer type.